lua/lv-which-key/init.lua

```diff
@@ -187,10 +148,24 @@ local mappings = {
   },
   T = {
     name = "Treesitter",
-    i = { ":TSConfigInfo<cr>", "Info" },
+    i = { "<cmd>TSConfigInfo<cr>", "Info" },
   },
 }
 
+local vopts = {
+  mode = "v", -- VISUAL mode
+  prefix = "<leader>",
+  buffer = nil, -- Global mappings. Specify a buffer number for buffer local mappings
+  silent = true, -- use `silent` when creating keymaps
+  noremap = true, -- use `noremap` when creating keymaps
+  nowait = false, -- use `nowait` when creating keymaps
+}
+-- NOTE: Prefer using : over in the command expression as the later avoids going back in normal-mode.
```

Comment:
Sorry, it probably cut text during my review, `-- NOTE: Prefer using : over <cmd>`
\<cmd> is missing

Reply:
Okay. Should we use : everywhere then? Or just for visual mappings?

Reply:
I set it to you : for the visual binding now, and <cmd> for the normal ones. Let me know if that's wrong, and I'll change it. Sorry about the confusion. I updated the not to be more clear too.